.github/workflows/terraform.yaml

```diff
@@ -1,22 +1,59 @@
 name: Validate Terraform
+
 on:
+  - push
   - pull_request
+
 jobs:
   terraform:
     name: 'Terraform'
+
     runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        subdir: [ephemeral-setup, one-time-setup, monitoring]
+
     steps:
-      - name: 'Checkout'
+      - name: Checkout
         uses: actions/checkout@master
-      - name: 'Terraform Init'
-        uses: hashicorp/terraform-github-actions@master
+
+      - name: Remove Backend Config
```

Comment:
i guess if we decide to remove the s3 backends we should just codify that everywhere right?

Reply:
This is a cheat predicated on the idea that we're keeping S3 backends but they were causing issues for configuring the validation.    So the cheat just removes the backend config from our source code and validates as-if we use default backend handling with no explicit config.